packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -60,22 +62,27 @@ export enum FlushMode {
 	TurnBased,
 }
 
+/**
+ * @remarks Note: this API is experimental and not ready for production use.
+ *
+ * @alpha
+ */
 export enum FlushModeExperimental {
 	/**
 	 * When in Async flush mode, the runtime will accumulate all operations across JS turns and send them as a single
 	 * batch when all micro-tasks are complete.
 	 *
 	 * This feature requires a version of the loader which supports reference sequence numbers. If an older version of
 	 * the loader is used, the runtime will fall back on FlushMode.TurnBased.
-	 *
-	 * @experimental - Not ready for use
```

Comment:
Note: `@experimental` is not a supported TSDoc tag. Applied the `@alpha` release tag to the entire enum.